gobblin-dataset-config/src/main/java/gobblin/config/client/ConfigClient.java

```diff
@@ -0,0 +1,20 @@
+package gobblin.config.client;
+
+import java.net.URI;
+import java.util.Collection;
+
+import com.typesafe.config.Config;
+
+import gobblin.config.configstore.ConfigStore;
+
+public interface ConfigClient {
+
+  /**
+   * @param cs
+   * @param uri - the uri relative to the input ConfigStore
+   * @return
+   */
+  public Config getConfig(ConfigStore cs, URI uri);
```

Comment:
I don't think ConfigStore should be a parameter, it should be inferred from the uri